### DISCUSSION TITLE
Default status screen section changed?

### DISCUSSION THREAD

**Discussion by gcv** [magit/magit#4539] [Support] [answered]
Just installed 3.2... has the default section in the status screen changed? Fresh `magit-status` buffers always used to open with the point at the top of the buffer for me, and now it opens with the point either on Unstaged changes or Recent commits, and the Head section off-screen. Is there a way to get the old behavior back (short of adding a hook with `(goto-line 1)`)?

PS: All commands are suddenly crazy slow also, but that's a separate investigation I haven't done yet.

**Comment by tarsius** [comment, marked as answer]
Sounds like you haven't updated in nearly three years, this changed in `2.90.0`. `(setq magit-status-initial-section nil)` restores the old behavior. When the status buffer is first created the default now is to jump to the first section after the headings. This should only cause the headings to be scrolled off-window if the jumped to section does not fit in the window.

**Comment by gcv** [comment]
Thanks, @tarsius. Something strange happened to my Emacs session. In a new status window, the headings were scrolled off-window right away. Screenshot attached. But I restarted Emacs, and cannot reproduce this behavior. Strange. Writing this off as a false alarm for now. Thanks again.

![Screen Shot 2021-11-01 at 13 59 36](https://user-images.githubusercontent.com/10327/139744323-d5f8f65e-1481-43ee-9f08-3689cdae5d03.png)

**Comment by snowman** [comment]
Here is what i used:

```lisp
(setq magit-status-initial-section
      '(
        ((unstaged) (status))
        ((staged) (status))
        ((untracked) (status))
        ((unpushed . "@{upstream}..") (status))
        ))
```
